x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/expandable_flyout/alert_details_left_panel_session_view_tab.cy.ts

```diff
@@ -49,10 +46,10 @@ describe.skip(
         .and('have.text', 'Session View');
 
       // TODO ideally we would have a test for the session view component instead
-      cy.get(DOCUMENT_DETAILS_FLYOUT_VISUALIZE_TAB_SESSION_VIEW_ERROR)
-        .should('be.visible')
-        .and('contain.text', 'Unable to display session view')
-        .and('contain.text', 'There was an error displaying session view');
+      // cy.get(DOCUMENT_DETAILS_FLYOUT_VISUALIZE_TAB_SESSION_VIEW_ERROR)
```

Comment:
is that meant to be commented out?

Reply:
that's because SEESION_VIEW_ERROR was removed [here](https://github.com/elastic/kibana/pull/166298/files#diff-c076fcf4d06f338cad5621901aa34efe5a650a237b79045a3837795c0d45e554L15). Interestingly it didn't throw any errors on that PR. Commenting out to preserve the test, but let me know if you have any suggestions!

Reply:
I would remove the commented line for now, the variable doesn't exist anymore and we might never add it back or add a different text...